frontend/src/features/notifications/components/NotificationInformationModal.tsx

```diff
@@ -112,56 +124,85 @@ const DemandCapacityNotificationView = ({ demandCapacityNotification, partners }
                 <FormLabel>Text</FormLabel>
```

Comment:
```suggestion
                <FormLabel>Text*</FormLabel>
```

Didn't see that last time. There should be an asteric to indicate the mandatoryness now.

Reply:
Sorry for missing that small detail, thanks for catching it! :)

Reply:
Thanks for the further consistent changes! While walking through the changes, I noticed that we should also add the asteric for the resolution modal. Please incroporate that, too :)